runeq/v2sdk/stream.py

```diff
@@ -0,0 +1,47 @@
+"""
+V2 SDK functionality to support Stream operations.
+
+"""
+from runeq import errors
+from . import client
+
+
+def get_stream(stream_id, **params):
+    """
+    Fetch stream data from the Stream API.
+
+    Args:
+        stream_id: Unique string identifier for a stream.
+
+        **params:
+            limit: Limits the number of points returned for the stream.
+            format: 'csv' or 'json'. Defaults to csv.
+    """
+    if not client._stream_client:
+        raise errors.InitializationError("Failed to initialize stream client.")
+
+    url = f"{client._stream_client._config.stream_url}/v2/streams/{stream_id}"
+    return client._stream_client._get(url, **params)
+
+
+def get_availability(stream_id, **params):
+    """
+    Fetch stream data availability from the Stream API.
+
+    Args:
+        stream_id: Unique string identifier for a stream.
+
+        **params:
+            limit: Limits the number of points returned for the stream.
+            format: 'csv' or 'json'. Defaults to csv.
+    """
+    if not client._stream_client:
+        raise errors.InitializationError("Failed to initialize stream client.")
+
+    stream_url = client._stream_client._config.stream_url
+    url = f"{stream_url}/v2/streams/{stream_id}/availability"
+
+    # TODO: Currently it seems that only JSON availability is supported, but
```

Comment:
CSV should be supported (the same as the streams endpoint: with CSV by default and JSON if you pass a `format=json` param in the querystring). The linked ticket is about supporting batch availability (where you can ask for the availability for multiple streams at a time).